docs/install/pubsub-service-account.md

```diff
@@ -47,25 +47,40 @@ Service Account.
           --member=serviceAccount:cre-pubsub@$PROJECT_ID.iam.gserviceaccount.com \
           --role roles/pubsub.editor
         ```
-
-    1.  Download a new JSON private key for that Service Account. **Be sure not
+1. Access Google Cloud service. There are two methods provided to access 
+    Google Cloud service with [Google Cloud Service Account](https://console.cloud.google.com/iam-admin/serviceaccounts/project),
+    you can either:
+    1. Use workload identity.
+     
+        It is the recommended way to access Google Cloud services from within GKE due to its improved security properties and 
+        manageability. More information about [workload identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity). 
+        
+        1. Enable workload identity. Skip this step if it is already enabled in control plane.
```

Comment:
everywhere you use `workload identity` in docs, change it to `Workload Identity`

Reply:
`Skip this step if you already enable it in the [control plane setup](....)`. and put the link to the control plane installation setup